core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -26,28 +26,34 @@ import { ExplorationOpportunity } from '../opportunities-list-item/opportunities
 import constants from 'assets/constants';
 import { Subscription } from 'rxjs';
 
+type LoadOpportunities = Promise<{
+  opportunitiesDicts: ExplorationOpportunity[]; more: boolean;
```

Comment:
Are these opportunities or opportunitiesDicts? The former should be what's used to name domain objects (which is suggested by the typing here) and the latter should only be used for dicts with keys and values. Probably worth standardizing to avoid confusion.

Reply:
Actually we are using `opportunitiesDicts` in this whole file (may be mistake done by former developer who edit this file).
Now change to `opportunities`.